Yank/analyze.py

```diff
@@ -604,9 +604,20 @@ def extract_trajectory(output_path, nc_path, state_index=None, replica_index=Non
     try:
         nc_file = netcdf.Dataset(nc_path, 'r')
 
+        # Extract topology and system serialization
```

Comment:
Yeah, it felt weird to deserialize the system only to know if it's periodic, but it only takes few seconds (the extraction of the trajectory takes in the order of minutes), so I didn't think that changing the `.nc` format would be worth.